src/main/java/org/jitsi/videobridge/cc/vp8/VP8QualityFilter.java

```diff
@@ -382,6 +384,11 @@ private static int getSpatialLayerId(int index)
      * Gets a JSON representation of the parts of this object's state that
      * are deemed useful for debugging.
      */
+    @SuppressWarnings("unchecked")
+    @SuppressFBWarnings(
+            value = "IS2_INCONSISTENT_SYNC",
+            justification = "We intentionally avoid synchronizing while reading" +
```

Comment:
Reading through this class, I find the synchronization we're doing a bit confusing (I probably haven't parsed it fully) but I wonder if it's worth a look.

Reply:
I don't know whether the syncronization in the rest of the class makes sense, but for `getDebugState` my intention is as described: read the int fields without aquiring any locks to reduce the risk of a deadlock (since this runs in a completely different context)